src/libraries/externals.csproj

```diff
@@ -21,23 +18,23 @@
   </PropertyGroup>
 
   <Import Project="$(RepositoryEngineeringDir)coredistools.targets" Condition="$(CopyCoreDisToolsToCoreRoot)" />
-  
+
   <ItemGroup Condition="'$(DotNetBuildFromSource)' != 'true'">
     <PackageReference Include="Microsoft.DiaSymReader.Native"
                       Version="$(MicrosoftDiaSymReaderNativeVersion)" />
   </ItemGroup>
 
-  <ItemGroup Condition="'$(TargetsMobile)' != 'true' and '$(UseLiveBuiltDotNetHost)' != 'true'">
-    <PackageReference Include="Microsoft.NETCore.DotNetHost"
-                      Version="$(MicrosoftNETCoreDotNetHostVersion)" />
-    <PackageReference Include="Microsoft.NETCore.DotNetHostPolicy"
-                      Version="$(MicrosoftNETCoreDotNetHostPolicyVersion)" />
```

Comment:
These are not used anywhere else, we can delete them from version files:
https://github.com/dotnet/runtime/blob/17a4eca42337109c4f916a023d97df12b50b5d59/eng/Versions.props#L77-L78
https://github.com/dotnet/runtime/blob/17a4eca42337109c4f916a023d97df12b50b5d59/eng/Version.Details.xml#L213-L220